<launch>

  <arg name="robotname" default="robot_0" />

  <group ns="$(arg robotname)">
    <param name="robot_name" value="$(arg robotname)" />
    <param name="tf_prefix" value="$(arg robotname)" />
    
    <node pkg="laser_analysis" name="laserobstacle" type="laserobstacle" args="-gui"  output="screen">
	    <!-- remap from="scan" to="/$(arg robotname)/scan" / -->
	    <param name="laser_frame_id" value="/$(arg robotname)/laser_frame" />
	    <param name="map_distance" value="0.5" />
        <param name="obstacle_margin_x_min" value="0" />
        <param name="obstacle_margin_x_max" value="6" />
        <param name="obstacle_margin_y_min" value="-1" />
        <param name="obstacle_margin_y_max" value="1" />
    </node>
      
  </group>
  
</launch>
